src/Domain/CoordinateMaps/CubicScale.hpp

```diff
@@ -0,0 +1,93 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <limits>
+#include <string>
+#include <unordered_map>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Utilities/TypeTraits.hpp"
+/// \cond
+class FunctionOfTime;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace CoordMapsTimeDependent {
+/*!
+ * \ingroup CoordMapsTimeDependentGroup
+ * \brief CubicScale map defined by \f$x = a(t)*\xi+(b(t)-a(t))\xi^3/X^2\f$,
+ *  where \f$X\f$ is the outer boundary.
```

Comment:
From looking at the test, it looks like this map isn't necessarily a map from logical coordinates to physical coordinates. That is, here it not unreasonable to consider values for xi outside [-1, 1]. Could you add a little more detail as to where this kind of map might be used? Because currently most coordinate maps are for mapping a logical cube to some physical domain for the purposes of being used by Blocks in a DomainCreator, but as we add different kinds of maps it would be nice to have more information as to what the typical use of this map is.

Reply:
Additional documentation for maps is currently stuck in a concurrent pull request.
But you raise a good point, perhaps we should decide on a way to distinguish 
between the maps that are used for for logical to physical from those that are used by the control system in mapping from the grid (what you refer to as the physical domain, I think?) to some other frame (such as the distorted frame or the inertial frame).